### PR TITLE
JAVA-26758: Changes made for Upgrade openapi-generator-maven-plugin t…

### PR DESCRIPTION
**Pull request by Bipinkumar27**
…o latest version

### CODE REVIEW

File: lombok-modules/lombok-2/pom.xml
Comment (anastasiosioannidis) [inline lombok-modules/lombok-2/pom.xml:1]: Try removing io.swagger swagger-annotations.
I think it's not needed

Comment (Bipinkumar27) [reply lombok-modules/lombok-2/pom.xml:1]: let me check

Comment (Bipinkumar27) [reply lombok-modules/lombok-2/pom.xml:1]: Hey @anastasiosioannidis 

We need to have this with the upgrade version when generating the book model. The @Schema annotation is utilized for schema, and we require the io.swagger swagger-annotations.

Comment (anastasiosioannidis) [reply lombok-modules/lombok-2/pom.xml:1]: Ok

Comment (Bipinkumar27) [reply lombok-modules/lombok-2/pom.xml:1]: **_removing jackson-databind-nullable we are getting below error_** 

[ERROR] /Users/sw/Documents/Baeldung/baeldung-tutorials/lombok-modules/lombok-2/target/generated-sources/openapi/src/main/java/com/baeldung/openapi/model/Book.java:[7,41] package org.openapitools.jackson.nullable does not exist

_**removing validation-api.version we are getting the below error :**_

[ERROR] /Users/sw/Documents/Baeldung/baeldung-tutorials/lombok-modules/lombok-2/target/generated-sources/openapi/src/main/java/com/baeldung/openapi/model/Book.java:[10,1] package javax.validation.constraints does not exist

Comment (anastasiosioannidis) [reply lombok-modules/lombok-2/pom.xml:1]: @Bipinkumar27 

Regarding  jackson-databind-nullable and validation-api.version, I didn't say to remove these. They're good.

But please check and remove this one:

        <dependency>
            <groupId>io.swagger</groupId>
            <artifactId>swagger-annotations</artifactId>
            <version>${swagger.annotation}</version>
        </dependency>

**Pull request merged**